canonical-web-and-design/maas-ui
# Fix machine list checkboxes highlighting when action cannot be performed.

## Done
- Fixed Vanilla regression in which checkboxes would turn yellow if the action could not be performed on the selected machine.

## QA
- Select a bunch of machines that can't all perform an action.
- Select that action and see that the warning to update selection shows up, and the machines that can't perform the action have a yellow checkbox instead of a blue one. The group(s) and header checkboxes should also be yellow if the machine inside can't perform the action.

## Fixes
Fixes #871 

## Screenshot
![0 0 0 0_8400_MAAS_ (7)](https://user-images.githubusercontent.com/25733845/76300221-a9c73580-62bc-11ea-9f31-571f6564459d.png)
